src/Microsoft.OData.Core/Buffers/BufferUtils.cs

```diff
@@ -0,0 +1,37 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="BufferUtils.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+namespace Microsoft.OData.Buffers
+{
+    using System;
+
+    /// <summary>
+    /// Helpers to deal with buffers
+    /// </summary>
+

```

Comment:
nit: our coding style does not contain empty spaces between summary comment and class declaration.